diffulex/strategy/fast_dllm_v2/engine/request.py

```
@@ -0,0 +1,407 @@
+class FastDLLMV2Mode(IntEnum):
+    FULL_BUFFER_INIT = 0
+    SUB_BLOCK_REFINE = 1
+    FINAL_COMMIT = 2
+
+
+@AutoReq.register("fast_dllm_v2")
+class FastDLLMV2Req(DllmReq):
+    def __init__(
+        self,
+        token_ids: list[int],
+        sampling_params: SamplingParams = SamplingParams(),
```

Ruff (0.15.18)

[warning] Do not perform function call `SamplingParams` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)